Alignment/OfflineValidation/data/lumiperFullRun2.txt

```diff
@@ -0,0 +1,1970 @@
+266150 0
```

Comment:
Just to make this clear... Why lumiperFullRun2.txt cannot stay here and lumiPerRun_Run2.txt can? This in general is a problem as there is no environmental variable directly pointing to the data repository - any unit test reading such a file would fail as "search-for-correct-base-dir"-mechanism is not implemented for any of the local config parsers.

Reply:
> > Proven not to be working here
> 
> that proves only that was not working for python. Are all codes that consume these files python scripts? Anyway large ASCII files do not belong in `cmssw` .

Yes, all scripts that process local configuration are python. Each validation type needs to take care for its environmental variables explicitly in `Alignment/OfflineValidation/python/TkAlAllInOneTool/<Validation>.py`

Reply:
Here my comment: https://github.com/cms-sw/cmssw/pull/38304/files#r952867893